src/shipping/components/ShippingZonePostalCodes/ShippingZonePostalCodes.tsx

```diff
@@ -60,23 +57,44 @@ const useStyles = makeStyles(
     }
   }),
   {
-    name: "ShippingZoneZipCodes"
+    name: "ShippingZonePostalCodes"
   }
 );
 
-const ShippingZoneZipCodes: React.FC<ShippingZoneZipCodesProps> = ({
-  data,
+const ShippingZonePostalCodes: React.FC<ShippingZonePostalCodesProps> = ({
   disabled,
   initialExpanded,
-  zipCodes,
-  onZipCodeDelete,
-  onZipCodeInclusionChange,
-  onZipCodeRangeAdd
+  initialInclusionType,
+  postalCodes,
+  onPostalCodeDelete,
+  onPostalCodeInclusionChange,
+  onPostalCodeRangeAdd
 }) => {
   const [expanded, setExpanded] = React.useState(initialExpanded);
+  const [inclusionType, setInclusionType] = React.useState(
+    initialInclusionType
+  );
   const intl = useIntl();
   const classes = useStyles({});
 
+  const onInclusionRadioChange = (event: React.ChangeEvent<any>) => {
+    const value = event.target.value;
+    setInclusionType(value);
+    onPostalCodeInclusionChange(value);
+  };
```

Comment:
It looks like you're keeping the exact same state twice. What happens if `initialInclusionType` prop changes and your state is out-of-sync? Parent's `initialInclusionType` state could be modified by another component/query outside this component. Consider if this is ever a case, perhaps getting rid off `inclusionType` internal state here would solve this issue. You could then change the prop name `initialInclusionType` => `inclusionType` and lift the state out of this component